apps/preview/nuxt/pages/showcases/Search/SearchBasic.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <form @submit.prevent="submit" ref="referenceRef" class="relative">
+    <label>
+      <SfInput ref="inputRef" v-model="inputModel" @focus="open" placeholder="Search">
```

Comment:
suggestion: I believe we can't have `div` (SfInput) inside `label` :(

Reply:
We show it as a way to go in our examples: https://docs.storefrontui.io/v2/react/components/input.html#input-with-icons

Do you suggest any solution? We do have a problem, because right now it's a bit awkward when you can click on the icon inside the input, but it didn't get focused.

Reply:
I removed the label [`5cd40b6`](https://github.com/vuestorefront/storefront-ui/pull/2630/commits/5cd40b6813342c64269166fc6d7749a89b1fc510). I think we should revisit this issue though. I'm going to create a follow-up ticket for this.